.github/workflows/program-candy-machine.yml

```diff
@@ -0,0 +1,70 @@
+name: Program Candy Machine
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+env:
+  CARGO_TERM_COLOR: always
+  SOLANA_VERSION: 1.8.5
+  RUST_TOOLCHAIN: stable
+
+jobs:
+  changes:
+    runs-on: ubuntu-latest
+    # Set job outputs to values from filter step
+    outputs:
+      code: ${{ steps.filter.outputs.code }}
+    steps:
+    - uses: actions/checkout@v2
+    # For pull requests it's not necessary to checkout the code
+    - uses: dorny/paths-filter@v2
+      id: filter
+      with:
+        filters: |
+          core:
+            - 'core/**'
+          package:
+            - 'nft-candy-machine/**'
+  build-and-test-candy-machine:
+    needs: changes
+    if: ${{ needs.changes.outputs.code == 'true' }}
```

Comment:
Needs to be `if: ${{ needs.changes.outputs.core == 'true' || needs.changes.outputs.package == 'true' }}`